src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -93,7 +93,7 @@ import { AlertWrapper } from './styleds' // mod
 // `
 
 function reportAnalytics(action: string, label?: string, value?: number) {
-  ReactGA.event({
+  reportEvent({
```

Comment:
There's no import here, it's breaking the build https://github.com/cowprotocol/cowswap/runs/6907436836?check_suite_focus=true

Reply:
I think I had changes in my local...  this is why it was not working 

<img width="533" alt="image" src="https://user-images.githubusercontent.com/2352112/174405880-024f8345-cc13-493a-b77d-8d2a0ca899e4.png">